package.json

```diff
@@ -51,80 +51,80 @@
   },
   "homepage": "https://github.com/Seneca-CDOT/telescope#readme",
   "dependencies": {
-    "@bull-board/api": "3.6.0",
+    "@bull-board/api": "3.7.0",
     "@bull-board/express": "3.7.0",
-    "@elastic/elasticsearch": "7.11.0",
-    "@elastic/elasticsearch-mock": "0.3.0",
-    "@wordpress/wordcount": "2.15.0",
+    "@elastic/elasticsearch": "7.15.0",
+    "@elastic/elasticsearch-mock": "0.3.1",
+    "@wordpress/wordcount": "3.2.3",
     "body-parser": "1.19.0",
```

Comment:
Filed #2504 to remove this.  We don't need it.

Reply:
You can't remove this withing fixing the other things I mention in #2504.  Please add `body-parser` back, and read review comments more carefully.